docs/content.zh/docs/connectors/flink-sources/postgres-cdc.md

```diff
@@ -470,7 +470,6 @@ public class PostgresParallelSourceExample {
                         .slotName("flink")
                         .decodingPluginName("decoderbufs") // use pgoutput for PostgreSQL 10+
                         .deserializer(deserializer)
-                        .includeSchemaChanges(true) // output the schema changes as well
```

Comment:
Seems this undocumented method still exists in `PostgresSourceExampleTest` and `org.apache.flink.cdc.connectors.postgres.source.PostgresSourceBuilder#includeSchemaChanges`. Should they be removed, too?

Reply:
> Seems this undocumented method still exists in `PostgresSourceExampleTest` and `org.apache.flink.cdc.connectors.postgres.source.PostgresSourceBuilder#includeSchemaChanges`. Should they be deprecated / removed, too?

Yes, I think it should be removed as well, thank you for reminding me